_pages/en/esdc-it-strategy.md

```diff
@@ -52,43 +45,114 @@ But a complex and uncertain world rewards an empirical cycle of trying, observin
+Because strategy making fosters long term impacts which are difficult to measure directly, the following IT Strategy Logic model helps visualize the causes and effects of output activities that are expected to directly affect short term outcomes and lead to long term impacts.
 
-![IM/IT services and key functions](assets/images/imit-services-and-functions.png "IM/IT Services and key functions")
+![ESDC's IT Strategy Logic Model - showing the short, medium, and long term outcomes](assets/images/it-strategy-logic-model.png "IT Strategy Logic Model")
 
-The goal of the ESDC IT Strategy is to improve each of those services and functions without telling their respective teams *what to do*.
-Instead, the IT Strategy focuses on empowering the teams to continuously improve by **investing in core competencies**.
+The above diagram positions ESDC's IT within its current state of transitioning to Digital Government at the same time as requiring to maintain its legacy systems until they are being replaced with modern ones. In addition, the [4 high level priorities of IITB](http://esdc.prv/en/iitb/corporate/Who_We_Are/Plans_and_Priorities/index.shtml) are listed to remind the focus of the organization.
 
-### Empowering IT teams
+### Input and Outputs
 
-The ESDC IT Strategy uses the [DevOps Research and Assessment (DORA)](https://cloud.google.com/devops) 21 capabilities as a benchmark to identify areas of improvements.
-The choice of technology is not what is important, what is important is to continuously increase frequency of delivery while reducing lead time, time to restore service when an incident occurs, rate of failure, and cyber security incidents.
+**Inputs** are what both the IT Strategy team require to produce their outputs.
+Inputs  for other IITB teams is not displayed here as a means to reduce redundancy and confusion. Other IITB teams play an essential role and have their distinct section (see below)
 
-The transition towards this partnership model is explained in the following [Strategy Map diagram](strategy-summary.html).
-Each block corresponding in the diagram represents a distinct strategy to move ESDC towards its ultimate Digital Transformation goal: become a key stakeholder in the Canadian Government as a Platform.
+**Outputs** are the IT Strategy team's own activities and creation.
+Those outputs are expected to create the short term outcomes listed below, which will lead to the long term impacts sought.
+
+### Short Term Outcomes
+
+As culture is defined by a set of behaviours, the short term outcomes sought by the ESDC IT Strategy is to create the environment needed to manifest them.
+More specifically, an environment that will produce:
+
+1) **Reduced IT project sizes**: Lines of business become more aware of the risks in using technology and so reduce their initiative sizes. Advance planning is shifting to iteration and adjusting course using empirical evidence from user feedback.
```

Comment:
IMO this is very optimistic as a short term outcome. Possibly reword to increase awareness of risk relating to large IT projects

Reply:
This awareness is what we will do. As an outcome, we want project sizes to reduce.
It doesn't mean all projects will be smaller in sizes, but we should see some results taking to that effect (i.e. small projects vs minor projects vs major projects).

Work by the PMO is promoting this as well, in parallel to our efforts. E.g. their PoC framework that LOBs can use as pre-project activity to inform themselves on the viability of an idea.

Reply:
will keep it as **reduced IT project sizes** :)